x/dao/keeper/msg_server_distribution_result.go

```diff
@@ -53,17 +58,21 @@ func (k msgServer) resolveStagedClaims(ctx sdk.Context, start int64, end int64)
 		keys = append(keys, p)
 	}
 	for _, p := range keys {
-		err = k.convertClaim(ctx, p, popParticipants[p])
+		err = k.convertAccountClaim(ctx, p, popParticipants[p])
 		if err != nil {
 			return err
 		}
 	}
 
+	// convert dao module claims
+	burnCoins, mintCoins := getConvertCoins(daoModuleAmt)
+	err = k.convertCoins(ctx, burnCoins, mintCoins)
+
```

Comment:
The coins cannot be converted (burned and minted) because they weren't minted in the first place. Please remove this.